services/movie.py

```diff
@@ -1,27 +1,34 @@
+from typing import Optional
+
+from django.db import transaction
 from django.db.models import QuerySet
 
 from db.models import Movie
 
 
 def get_movies(
-    genres_ids: list[int] = None,
-    actors_ids: list[int] = None,
+    genres_ids: Optional[list[int]] = None,
+    actors_ids: Optional[list[int]] = None,
+    title: Optional[str] = None
 ) -> QuerySet:
     queryset = Movie.objects.all()
+    if title:
+        return queryset.filter(title__contains=title)
 
     if genres_ids:
         queryset = queryset.filter(genres__id__in=genres_ids)
 
     if actors_ids:
         queryset = queryset.filter(actors__id__in=actors_ids)
 
-    return queryset
+        return queryset
```

Comment:
Remove leading spaces. Leave return at the same level as previously

Reply:
@svitlana-savikhina not fixed. Undo this changes
<img width="279" alt="image" src="https://user-images.githubusercontent.com/65166203/232057137-d7901405-abf0-4e1e-9ace-c81146f05d3c.png">